docs/guides/build-using-too-many-resources.rst

```diff
@@ -23,3 +23,20 @@ If there are extra packages that you don't need for building docs,
 you can create a custom requirements file just for documentation.
 This should speed up your documentation builds,
 as well as reduce your memory footprint.
+
+Use pip when possible
+---------------------
+
+In case you are using `conda` to create your environment,
+you could consider using `pip` instead since `conda` could require too much memory to calculate the dependency tree.
+
+Use system site-packages for pre-installed libs
+-----------------------------------------------
+
+There are a few libraries that Read the Docs has already installed (scipy, numpy, matplotlib, pandas, etc)
+in the Docker image used to build your docs. You can check the updated list of pre-installed libraries in the `Docker image repository`_.
```

Comment:
We probably need to document which exact libraries are available. I don't think it's reasonable for people to have to dig through a git repo for this data.

Reply:
Do we want to document this here or in the specific YAML config at https://github.com/rtfd/readthedocs.org/pull/4422 ?

Also, this depends on the Docker Image used itself so, I'm not sure that there is a generic way to document this and keep it updated. That's why I mentioned them generally and points to the repository.

Reply:
I think documenting this in the yaml config makes sense, or maybe on another _central doc_, because we'll be having multiple versions, and each with the same content about build images.